AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -318,7 +324,13 @@ export class SummaryComponent implements OnInit, OnDestroy {
                     this.logger.info(`${this.loggerPrefix} Updated booking. Navigating to booking details.`, {
                         hearingId: hearingDetailsResponse.id
                     });
-                    this.router.navigate([PageUrls.BookingDetails]);
+
+                    if (hearingDetailsResponse.status === BookingStatus.Failed.toString()) {
+                        this.hearing.hearing_id = hearingDetailsResponse.id;
+                        this.setError(`Failed to book new hearing for ${hearingDetailsResponse.created_by} `);
+                        return;
+                    }
+                    this.router.navigate([PageUrls.BookingConfirmation]);
```

Comment:
Is this redirect supposed to change?

Reply:
@oliver-scott Yes, the redirect is supposed to change